scripts/check_pipfile_and_toxini.py

```diff
@@ -39,8 +39,6 @@
     "tomte[safety]": "==0.1.5",
     "tomte[vulture]": "==0.1.5",
     "tomte[darglint]": "==0.1.5",
-    "open-aea[all]": "==1.21.0.post1",
-    "open-aea-cli-ipfs": "==1.21.0",
```

Comment:
why removed?

Reply:
I think the check was failing because we were installing `1.21.0.post1` in the tox env but the version for open-aea was `1.21.0` because we don't actually merge the branch with the post release